src/test/java/com/textadventure/MainTest.java

```diff
@@ -1,11 +1,35 @@
 package com.textadventure;
 
-import org.junit.Assert;
-import org.junit.Test;
-
-public class MainTest {
-    @Test
-    public void testMain() {
-        Assert.assertEquals(true, true);
-    }
-}
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+
+public class LexerTest {
+
+	@Test
+	void testOutputToken() {
+		String inputTest = "North run walk";
+		ArrayList<Token> tokens = Lexer.lex(inputTest);
+		String output = "";
+		for (Token token : tokens)
+			output += token.type + " ";
```

Comment:
Instead of `output += token.type + " ";` you could use **StringBuilder** to append strings together and to get the final output you can do sb.toString(). Looks a bit cleaner when dealing with string concatenation.